ledger/test/test_compact_serializer.py

```diff
@@ -0,0 +1,228 @@
+from collections import OrderedDict
+
+from ledger.serializers.compact_serializer import CompactSerializer
+
+fields = OrderedDict([
+    ("f1", (str, str)),
+    ("f2", (str, str)),
+    ("f3", (str, int)),
+    ("f4", (str, float)),
+    ("f5", (str, eval))
+])
+
+def testInitCompactSerializerWithCorrectFileds():
+    CompactSerializer(fields)
```

Comment:
It tests expected format of fields (contract). We need to have a class for Fields presentation (not just a dict). There is TODO comment for this.